core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkspaceComponent.scala

```diff
@@ -326,6 +326,11 @@ trait WorkspaceComponent {
     ): ReadAction[Option[Workspace]] =
```

Comment:
This is still getting used by `WorkspaceSupport.getWorkspaceContext` which is used by a bunch of entity and snapshot service things. I think we'll want to switch those to stop returning v2 workspaces as well.